internal/ui/src/components/empty.tsx

```diff
@@ -0,0 +1,68 @@
+import type React from "react";
+
+import { cn } from "../lib/utils";
+
+interface EmptyRootProps extends React.HTMLAttributes<HTMLDivElement> {
+  fill?: boolean;
+}
+export function Empty({ className, children, ...props }: EmptyRootProps) {
+  return (
+    <div
+      className={cn("animate-in fade-in-50 p-8 text-center h-full w-full", className)}
+      {...props}
+    >
+      <div className="flex flex-col items-center justify-center h-full w-full ">{children}</div>
```

Comment:
do we need this inner div?

it feels like we could just use 1 div with all classes